crates/db_connection_pool/src/postgrespool.rs

```diff
@@ -93,14 +106,32 @@ impl PostgresConnectionPool {
                     connection_string.push_str(format!("port={pg_port} ").as_str());
                 }
                 if let Some(pg_sslmode) = params.get("pg_sslmode") {
-                    connection_string.push_str(format!("sslmode={pg_sslmode} ").as_str());
+                    match pg_sslmode.to_lowercase().as_str() {
+                        "disable" | "require" | "prefer" => {
```

Comment:
Shouldn't we also support `verify-ca` and `verify-full`?

Reply:
No, we use `tokio-postgres` and it supports only require, prefer and disable options
https://github.com/sfackler/rust-postgres/blob/98f5a11bc0a8e451552d8941ffa078c7eb6cd60c/tokio-postgres/src/config.rs#L41-L51